data/json/items/comestibles.json

```diff
@@ -8144,6 +8144,94 @@
     },
     {
         "type" : "COMESTIBLE",
+        "id": "hickory_nut_unshelled",
+        "name": "handful of unshelled hickory nuts",
+        "name_plural": "handfuls of unshelled hickory nuts",
+        "weight" : 190,
+        "color" : "brown",
+        "addiction_type" : "none",
+        "spoils_in" : 0,
+        "stim" : 0,
+        "container" : "null",
+        "to_hit" : 0,
+        "comestible_type" : "FOOD",
+        "symbol" : "%",
+        "quench" : -5,
+        "healthy" : 1,
+        "addiction_potential" : 0,
+        "nutrition" : 15,
+        "description": "A handful of raw hard nuts from a hickory tree, their shells have been removed.",
+        "price" : 0,
+        "material" : "veggy",
+        "tool" : "null",
+        "volume" : 1,
+        "cutting" : 0,
+        "phase" : "solid",
+        "charges" : 2,
+        "bashing" : 0,
+        "fun" : 0
+    },
+    {
+        "type" : "COMESTIBLE",
+        "id" : "hickory_nut_roasted",
+        "name" : "handful of roasted hickory nut",
+        "name_plural" : "handfuls of roasted hickory nuts",
+        "weight" : 190,
+        "color" : "brown",
+        "addiction_type" : "none",
+        "spoils_in" : 0,
+        "stim" : 0,
+        "container" : "null",
+        "to_hit" : 0,
+        "comestible_type" : "FOOD",
+        "symbol" : "%",
+        "quench" : -5,
+        "healthy" : 1,
+        "addiction_potential" : 0,
+        "nutrition" : 15,
+        "description" : "A handful roasted nuts from a hickory tree.",
+        "price" : 90,
+        "material" : "veggy",
+        "tool" : "null",
+        "volume" : 1,
+        "cutting" : 0,
+        "phase" : "solid",
+        "charges" : 1,
+        "bashing" : 0,
+        "flags" : ["EATEN_HOT"],
+        "fun" : 2
+    },
+    {
+        "type" : "COMESTIBLE",
+        "id" : "hickory_nut_ambrosia",
+        "name" : "hickory nut ambrosia",
+        "weight" : 254,
+        "color" : "brown",
+        "addiction_type" : "none",
+        "spoils_in" : 259,
+        "stim" : 0,
+        "container" : "bottle_plastic",
+        "to_hit" : 0,
+        "comestible_type" : "DRINK",
+        "symbol" : "~",
+        "quench" : 40,
+        "healthy" : 0,
+        "addiction_potential" : 0,
+        "nutrition" : 20,
+        "description" : "Delicious hickory nut ambrosia. A drink worthy of the gods.",
+        "price" : 100,
+        "material" : ["veggy","milk"],
+        "tool" : "null",
+        "volume" : 1,
+        "cutting" : 0,
+        "phase" : "liquid",
+        "charges" : 1,
```

Comment:
At 1 charge per 4 nuts + some sugar, it is very wasteful when it comes to nutrition. Is it supposed to be?
Instead of having 10 fun, it could have 5 fun but produce 4 charges. That way it would produce more morale per craft anyway.

If you go with increased charges, lower the quench to 10 (because 4 charges * 10 = 40) and nutrition to 10 or 15.